docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/database-identifiers-shortened.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Database identifiers shortened in v5
+description: Database identifiers are shortened in Strapi v5 and can't be longer than 53 characters to avoid issues with identifiers that are too long.
+sidebar_label: Database identifiers shortened
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - database
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Database identifiers shortened in v5
+
+In Strapi v5, database identifiers can't be longer than 53 characters. <Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Database identifiers could be longer than 53 characters, potentially causing issues with some databases.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi v5**
+
+Database identifiers can't be longer than 53 characters and will be shortened.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+- A hashing key will be added when shortening database identifiers to avoid conflicts. It will consist in the first 6 characters of SHA-256. For example, `my_very_very_very_very_very_very_very_too_long_identifier_unique` will be shortened to `my_very_very_very_very_very_very_very_very_a2d3x3_unq` in Strapi v5.
```

Comment:
@Marc-Roig (if it was you writing/developing the database changes 🤔):
I fail to understand how we get the `a2d3x3` part, even using tools such as https://emn178.github.io/online-tools/sha256.html. Could you please elaborate so I can explain it a bit better to users? 😊 
![Screenshot 2023-10-26 at 14 44 41](https://github.com/strapi/documentation/assets/4233866/bf8f67c0-78ca-4ebe-95f1-abe82fd4e3b5)

Reply:
I think it was probably alex who wrote this piece of text. I bet that encoding is not accurate with sha-256, and it was jus tmade up for the example. it would be a good idea to actualy edit it with the real encoding!

Reply:
Thank you, Marc!